src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -156,6 +157,14 @@ class ExplorerButtonGroup extends React.Component {
       }
       clickFunc = this.exportPFBToURL(buttonConfig.targetURLTemplate);
     }
+    if (buttonConfig.type === 'export-file-pfb-to-url') {
+      if (!buttonConfig.targetURLTemplate) {
+        throw new Error('Misconfiguration Error! Expected button of type `export-pfb-to-url` to have the required `targetURLTemplate` property');
+      } else if (buttonConfig.targetURLTemplate.indexOf(PRESIGNED_URL_TEMPLATE_VARIABLE) === -1) {
+        throw new Error(`Misconfiguration error! An \`export-pfb-to-url\` button has a bad \`targetURLTemplate\` property. The string \`${PRESIGNED_URL_TEMPLATE_VARIABLE}\` must appear in the \`targetURLTemplate\` property. Bad \`targetURLTemplate\`: ${this.state.targetURLTemplate}`);
```

Comment:
I suppose we should mention `export-file-pfb-to-url` in these error messgaes?

Reply:
fixed!